### PR TITLE
Fix the docs of eventwatcher

### PR DESCRIPTION
**Pull request by knanao**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs-dev/user-guide/event-watcher.md [hunk 1]
```diff
@@ -18,8 +18,7 @@ While it empowers you to build pretty versatile workflows, the canonical use cas
 This guide walks you through configuring Event watcher and how to push an Event.
 
 ## Prerequisites
-If you want to use EventWatcher configuration files under the `.pipe/` directory, before we get into configuring EventWatcher, be sure to configure Piped. See [here](/docs/user-guide/managing-piped/configuring-event-watcher/) for more details.
-In other cases, there is no prerequisite. Let's move on to Usage.
+Before we get into configuring EventWatcher, be sure to configure Piped. See [here](/docs/user-guide/managing-piped/configuring-event-watcher/) for more details.
```
Comment (khanhtc1202) [inline docs/content/en/docs-dev/user-guide/event-watcher.md:21]: This is only true in case of using `.pipe/` pattern, isn't it?

Comment (knanao) [reply docs/content/en/docs-dev/user-guide/event-watcher.md:21]: Sorry, we need to configure `repoID` hence fixed like this.

File: docs/content/en/docs-dev/user-guide/managing-piped/configuring-event-watcher.md [hunk 2]
```diff
@@ -27,6 +27,8 @@ spec:
 ```
 
 ### [optional] Specify Eventwatcher files Piped will use
+>NOTE: This way is valid only for defining events using `.pipe/`.
```
Comment (khanhtc1202) [inline docs/content/en/docs-dev/user-guide/managing-piped/configuring-event-watcher.md:30]: should we add a link to the docs to this `.pipe/` pattern. I feel the docs are a bit complicated here.

**Review by khanhtc1202 — APPROVED** [review]
Nice improvement 👍

**Pull request merged**